package.json

```diff
@@ -3,18 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.2",
+    "apollo-boost": "^0.4.4",
     "apollo-server-lambda": "2.9.3",
     "encoding": "0.1.12",
-    "graphql": "14.5.4",
+    "graphql": "^14.5.4",
     "graphql-tag-pluck": "0.8.4",
     "merge-graphql-schemas": "1.7.0",
     "netlify-lambda": "1.6.3",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.1.1"
   },
   "scripts": {
     "start": "react-scripts start",
+    "start:lambda": "netlify-lambda serve src/lambda",
```

Comment:
This is no longer needed if we are using [Netlify Dev](https://github.com/netlify/cli/blob/master/docs/netlify-dev.md) as it will handle auto-detecting we are using `netlify-lambda` and properly build our functions.

Reply:
Thats cool! I didn't know about that.